src/current/releases/index.md

```diff
@@ -189,7 +189,11 @@ The following releases and their descriptions represent proposed plans that are
     skippable: {{ skippable }}<br /><br />
 {% endif %}
 
-CockroachDB {{ v.major_version }} is {% if skippable == true %}an [Innovation release]({% link releases/release-support-policy.md %}#innovation-releases) that is optional for CockroachDB {{ site.data.products.dedicated }} and CockroachDB {{ site.data.products.core }} but required for CockroachDB {{ site.data.products.serverless }}.{% else %}a required [Regular release]({% link releases/release-support-policy.md %}#regular-releases).{% endif %}{% if released == false %} It is still in development and not yet supported.{% endif %}{% unless latest_full_production_version.release_name != v.major_version %} CockroachDB {{ latest_full_production_version.release_name }} is the latest supported version.{% endunless %} To learn more, refer to [CockroachDB {{ latest.major_version }} Release Notes]({% link releases/{{ v.major_version }}.md %}).
+{% if released == false %}
+CockroachDB {{ page.major_version }} is in active development and is not yet supported. The following [testing releases]({% link releases/index.md %}#release-types) are intended for testing and experimentation only, and are not qualified for production environments or eligible for support or uptime SLA commitments. When CockroachDB {{ page.major_version }} is Generally Available (GA), production releases will also be announced on this page.
```

Comment:
```suggestion
CockroachDB {{ page.major_version }} is in active development and is not yet supported. The following [testing releases]({% link releases/index.md %}#patch-releases) are intended for testing and experimentation only, and are not qualified for production environments or eligible for support or uptime SLA commitments. When CockroachDB {{ page.major_version }} is Generally Available (GA), production releases will also be announced on this page.
```

There's a lot to get through in the subheading `Release types > Major releases` which is not really relevant, so I figure it's better to link directly to `Release types > Patch releases`, which has the definitions of Production and Testing releases.

Reply:
This is the main fix in this PR -- previously v24.3 was reporting that it was GA because of a bug. This fixes that bug and puts back the legal limiting languages about test releases that was already there. I don't want to imply on a What's New page for a testing release that it is supported in any way, so I don't think this PR is the right place to remove this. This is the place where they are downloading a Testing release and reading its release notes. WDYT?

Reply:
This is just a suggested update to the anchor on the link. Assuming you misinterpreted, please take another look? (Or let me know if I'm missing anything.) TY